build/checkOrphanedImages.js

```diff
@@ -0,0 +1,91 @@
+const path     = require('path');
+const fs       = require('fs');
+const common   = require('../scripts/common');
+const {
+  prompt,
+  flatten,
+  fetchFiles } = require('./utils/cli');
+
+const ignored_files            = [];
+const ignored_image_extensions = [
+    'DS_Store', 'zip',
+];
+
+const ignored_language_directories = [
```

Comment:
@easteregg I would be better to import and use https://github.com/binary-com/binary-static/blob/44b227ac570a1d24db7ce567a7b6bb2b7d6194f8/scripts/common.js#L36